Essentials/src/main/java/com/earth2me/essentials/Essentials.java

```diff
@@ -490,7 +542,7 @@ public void reload() {
     @Override
     public List<String> onTabComplete(final CommandSender sender, final Command command, final String commandLabel, final String[] args) {
         return onTabCompleteEssentials(sender, command, commandLabel, args, Essentials.class.getClassLoader(),
-            "com.earth2me.essentials.commands.Command", "essentials.", null);
+                "com.earth2me.essentials.commands.Command", "essentials.", null);
```

Comment:
```suggestion
            "com.earth2me.essentials.commands.Command", "essentials.", null);
```
Unnecessary spacing change